packages/react/src/component-tests/IcCard/IcCard.cy.tsx

```diff
@@ -0,0 +1,760 @@
+/// <reference types="Cypress" />
+
+import React from "react";
+import {
+  IcBadge,
+  IcButton,
+  IcCard,
+  IcStatusTag,
+  IcTypography,
+} from "../../components";
+import {
+  BE_VISIBLE,
+  HAVE_CLASS,
+  HAVE_CSS,
+  HAVE_FOCUS,
+  HAVE_LENGTH,
+  NOT_HAVE_CLASS,
+  NOT_HAVE_FOCUS,
+} from "../utils/constants";
+import { mount } from "cypress/react";
+import { SlottedSVG } from "../../react-component-lib/slottedSVG";
+
+const DEFAULT_TEST_THRESHOLD = 0.03;
+
+describe("IcCard", () => {
+  beforeEach(() => {
+    cy.injectAxe();
+  });
+  afterEach(() => {
+    cy.task("generateReport");
+  });
+
+  it("renders Card", () => {
+    mount(
+      <div style={{ margin: "16px" }}>
+        <a href="/">
+          <IcCard heading="Card" message="This is a static card"></IcCard>
+        </a>
+      </div>
+    );
+    cy.checkHydrated("ic-card");
+    cy.get("a").should(HAVE_CLASS, "ic-card-wrapper-link");
+    cy.get("a").focus().should(HAVE_FOCUS);
+    cy.checkA11yWithWait();
+  });
+
+  it("should click on card", () => {
+    mount(
+      <div style={{ margin: "16px" }}>
+        <IcCard
+          heading="Americano order"
+          subheading="Name: Michael"
+          message="Extras: Double espresso shot and oat milk."
+          clickable
+          onClick={(ev) => console.log(ev)}
+        >
+          <SlottedSVG
+            slot="icon"
+            xmlns="http://www.w3.org/2000/svg"
+            height="24px"
+            viewBox="0 0 24 24"
+            width="24px"
+            fill="#000000"
+          >
+            <path d="M2,21V19H20V21H2M20,8V5H18V8H20M20,3A2,2 0 0,1 22,5V8A2,2 0 0,1 20,10H18V13A4,4 0 0,1 14,17H8A4,4 0 0,1 4,13V3H20M16,5H6V13A2,2 0 0,0 8,15H14A2,2 0 0,0 16,13V5Z" />
+          </SlottedSVG>
+        </IcCard>
+      </div>
+    );
+    cy.checkHydrated("ic-card");
+    cy.get("ic-card").should(BE_VISIBLE).click().should(HAVE_FOCUS);
+    cy.checkA11yWithWait();
+  });
+
+  it("should get focussed class applied when parent link focussed", () => {
```

Comment:
Still an issue with this test, it now focuses the child and tests if it has focus, rather than focus the parent and check if the child has the focus class, currently it's just testing focus which is covered elsewhere.

Reply:
Unfortunately there's nowhere in the code that the card-title gets a focus class, so we can't test that. Instead we've tested that the card-title gets the correct styling for text-decoration on focus.